_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -8831,6 +8899,21 @@
 /obj/structure/table/abductor,
 /turf/open/floor/plating/abductor,
 /area/centcom/holding/cafe)
+"nKE" = (
+/obj/structure/table/wood,
+/obj/item/stack/sheet/mineral/gold{
+	amount = 5
+	},
+/obj/item/stack/sheet/mineral/silver{
+	amount = 5
+	},
+/obj/item/stack/sheet/mineral/plasma/thirty{
```

Comment:
/obj/item/stack/sheet/mineral/plasma/
not /obj/item/stack/sheet/mineral/plasma/thirty